src/bidiMapper/domains/context/cdpTarget.ts

```diff
@@ -92,13 +98,33 @@ export class CdpTarget {
     return this.#cdpClient;
   }
 
-  /**
-   * Needed for CDP escape path.
-   */
+  /** Needed for CDP escape path. */
   get cdpSessionId(): Protocol.Target.SessionID {
     return this.#cdpSessionId;
   }
 
+  /** Calls `Fetch.enable` with the added network intercepts. */
+  async fetchEnable() {
+    await this.#cdpClient.sendCommand(
+      'Fetch.enable',
+      this.#networkStorage.getFetchEnableParams()
+    );
+  }
+
+  /** Calls `Fetch.disable`. */
+  async fetchDisable() {
+    await this.#cdpClient.sendCommand('Fetch.disable');
+  }
+
+  /**
+   * Calls `Fetch.disable` followed by `Fetch.enable`.
```

Comment:
Can we have a line explaining why we need this.

Reply:
Done